data/blog/open-source-log-management.mdx

```diff
@@ -1,21 +1,25 @@
 ---
 title: 7 Open-Source Log Management Tools that you may consider in 2024
 slug: open-source-log-management
-date: 2024-05-13
+date: 2024-05-24
 tags: [Tools Comparison]
 authors: [daniel]
 description: Top open source log management tools in 2024 1.SigNoz 2.Graylog 3.Logstash 4.FluentD 5.Syslog-ng...
 hide_table_of_contents: false
 image: /img/blog/2023/01/open-source-log-management-cover.jpeg
-keywords: [logs,log management,log tools,open source log tools]
+keywords:
+  - logs
+  - log management
+  - log tools
+  - open source log tools
```

Comment:
hey @FavourDaniel this format wont work, it should be in array form only

Reply:
Fixed